<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Four In A line</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonJogar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>COMEÃ‡AR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTabuleiro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>41</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="pushButtonPronto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INSIRA OS JOGADORES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlayer1">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>80</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Jogador 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlayer2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Jogador 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRestart">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Jogar outra partida</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNexte">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelJogador1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelJogador2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbandonarPartida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Abandonar partida</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
